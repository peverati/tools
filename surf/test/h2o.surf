verbose
load h2o.wfn
surface
  epsiscp 0.220
  lebedev 5810
endsurface
