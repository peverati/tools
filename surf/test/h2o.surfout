 #  =====================================================
 # |            surf: Compute QTAIM surfaces             |
 #  =====================================================
 #
 # Calculation starts at 2019.04.23, 19:40:25.214
 # *** Begin to initialize data
 # *** End of initialize data
 # +++ Begin to read input
 # *** Verbose mode enabled ***
 # *** Reading data from h2o.wfn
 # Number of centers: 3
 # 1 O   8.0     0.000000    0.000000    0.221665
 # 2 H   1.0     1.430901    0.000000   -0.886660
 # 3 H   1.0    -1.430901    0.000000   -0.886660
 # *** Begin screeaning of shells
 # Number of molecular orbitals: 5
 # Input primitives 41 reduced to 41
 # CUTOFF for GTOs, eps:  0.1000000000E-13
 # CENTER 1
 # S Shell   Exp =   0.11720000E+05    Cutoff =      0.100000    Primitives :  1
 # S Shell   Exp =   0.17590000E+04    Cutoff =      0.200000    Primitives :  2
 # S Shell   Exp =   0.40080000E+03    Cutoff =      0.300000    Primitives :  3
 # S Shell   Exp =   0.11370000E+03    Cutoff =      0.600000    Primitives :  4
 # S Shell   Exp =   0.37030000E+02    Cutoff =      1.000000    Primitives :  5
 # S Shell   Exp =   0.13270000E+02    Cutoff =      1.600000    Primitives :  6
 # S Shell   Exp =   0.50250000E+01    Cutoff =      2.600000    Primitives :  7
 # S Shell   Exp =   0.10130000E+01    Cutoff =      5.700000    Primitives :  8
 # S Shell   Exp =   0.30230000E+00    Cutoff =     10.400000    Primitives :  9
 # P Shell   Exp =   0.17700000E+02    Cutoff =      1.400000    Primitives :  10 11 12
 # P Shell   Exp =   0.38540000E+01    Cutoff =      3.000000    Primitives :  13 14 15
 # P Shell   Exp =   0.10460000E+01    Cutoff =      5.700000    Primitives :  16 17 18
 # P Shell   Exp =   0.27530000E+00    Cutoff =     11.300000    Primitives :  19 20 21
 # D Shell   Exp =   0.11850000E+01    Cutoff =      5.500000    Primitives :  22 23 24 25 26 27
 # CENTER 2
 # S Shell   Exp =   0.13010000E+02    Cutoff =      1.600000    Primitives :  28
 # S Shell   Exp =   0.19620000E+01    Cutoff =      4.100000    Primitives :  29
 # S Shell   Exp =   0.44460000E+00    Cutoff =      8.600000    Primitives :  30
 # S Shell   Exp =   0.12200000E+00    Cutoff =     16.300000    Primitives :  31
 # P Shell   Exp =   0.72700000E+00    Cutoff =      6.900000    Primitives :  32 33 34
 # CENTER 3
 # S Shell   Exp =   0.13010000E+02    Cutoff =      1.600000    Primitives :  35
 # S Shell   Exp =   0.19620000E+01    Cutoff =      4.100000    Primitives :  36
 # S Shell   Exp =   0.44460000E+00    Cutoff =      8.600000    Primitives :  37
 # S Shell   Exp =   0.12200000E+00    Cutoff =     16.300000    Primitives :  38
 # P Shell   Exp =   0.72700000E+00    Cutoff =      6.900000    Primitives :  39 40 41
 # Total number of shells: 24
 # Rmaxatom for the atoms
 # 1  20.00
 # 2  20.00
 # 3  20.00
 # 24 shells contribute to the basin of center 1
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 # 24 shells contribute to the basin of center 2
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 # 24 shells contribute to the basin of center 3
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 # *** End of screeaning of shells
 # Occupied eps:  0.100000E-05
 # Number of occupied orbitals: 5
 # Number of virtual orbitals: 0
 # *** Reading surface options ***
 # *** Variable epsiscp changed to :  0.220000E+00
 # *** Surface lebedev changed to : 5810
 # *** End reading surface options ***
 # *** Computing surfaces
 # Will be computed for the following atoms
 # Number of surfaces : 3
 # 1 1
 # 2 2
 # 3 3
 # Rmaxsurf for the atoms
 # 1  10.00
 # 2  10.00
 # 3  10.00
 # Surface precision :  0.100000E-04
 # EPSISCP parameter :  0.220000E+00
 # Ntrial : 11
 # Rprimer :  0.400000E+00
 # Follow integration parameters
 # Atom 1 lebedev points 5810
 # Atom 2 lebedev points 5810
 # Atom 3 lebedev points 5810
 # Assuming nuclei 1 position: check!
 # Assuming nuclei 2 position: check!
 # Assuming nuclei 3 position: check!
 # Computing SURFACE for atom 1
 # Elapsed seconds :     11.21425
 # Computing SURFACE for atom 2
 # Elapsed seconds :     11.87365
 # Computing SURFACE for atom 3
 # Elapsed seconds :     11.87192
 # +++ End of read input
 #
 # Total elapsed time =       35.023261
 # Check : (0 WARNINGS, 0 COMMENTS)
 # Calculation ends at 2019.04.23, 19:41:00.237
