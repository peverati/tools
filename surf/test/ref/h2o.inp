h2o.wfn
epsiscp 0.220
lebedev 5810
