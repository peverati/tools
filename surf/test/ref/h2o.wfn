From PySCF
GAUSSIAN              5 MOL ORBITALS     41 PRIMITIVES        3 NUCLEI
  O    1    (CENTRE  1)   0.00000000  0.00000000  0.22166487  CHARGE = 8.0
  H    2    (CENTRE  2)   1.43090062  0.00000000 -0.88665950  CHARGE = 1.0
  H    3    (CENTRE  3)  -1.43090062  0.00000000 -0.88665950  CHARGE = 1.0
CENTRE ASSIGNMENTS    1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1
CENTRE ASSIGNMENTS    1  1  1  1  1  1  1  2  2  2  2  2  2  2  3  3  3  3  3  3
CENTRE ASSIGNMENTS    3
TYPE ASSIGNMENTS      1  1  1  1  1  1  1  1  1  2  2  2  3  3  3  4  4  4  2  3
TYPE ASSIGNMENTS      4  5  8  9  6 10  7  1  1  1  1  2  3  4  1  1  1  1  2  3
TYPE ASSIGNMENTS      4
EXPONENTS  1.1720000E+04 1.7590000E+03 4.0080000E+02 1.1370000E+02 3.7030000E+01
EXPONENTS  1.3270000E+01 5.0250000E+00 1.0130000E+00 3.0230000E-01 1.7700000E+01
EXPONENTS  3.8540000E+00 1.0460000E+00 1.7700000E+01 3.8540000E+00 1.0460000E+00
EXPONENTS  1.7700000E+01 3.8540000E+00 1.0460000E+00 2.7530000E-01 2.7530000E-01
EXPONENTS  2.7530000E-01 1.1850000E+00 1.1850000E+00 1.1850000E+00 1.1850000E+00
EXPONENTS  1.1850000E+00 1.1850000E+00 1.3010000E+01 1.9620000E+00 4.4460000E-01
EXPONENTS  1.2200000E-01 7.2700000E-01 7.2700000E-01 7.2700000E-01 1.3010000E+01
EXPONENTS  1.9620000E+00 4.4460000E-01 1.2200000E-01 7.2700000E-01 7.2700000E-01
EXPONENTS  7.2700000E-01
MO  1                     OCC NO =   2.00000000 ORB. ENERGY = -20.55053837
  5.69917765E-01  1.05873994E+00  1.77695558E+00  2.60027765E+00  3.02781798E+00
  2.22250309E+00  6.47733026E-01  1.20528976E-02 -7.75438109E-04  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
 -5.71444768E-03 -4.52290596E-03 -1.96905438E-03  0.00000000E+00  0.00000000E+00
  1.37972894E-04  2.51071130E-04  0.00000000E+00  0.00000000E+00 -2.75407791E-04
  0.00000000E+00  2.43366607E-05 -3.90400858E-05 -6.62213945E-05 -7.53714393E-05
  6.19988741E-05  5.45550935E-04  0.00000000E+00 -4.60467489E-04 -3.90400858E-05
 -6.62213945E-05 -7.53714393E-05  6.19988741E-05 -5.45550935E-04  0.00000000E+00
 -4.60467489E-04
MO  2                     OCC NO =   2.00000000 ORB. ENERGY =  -1.33644816
 -1.16480470E-01 -2.21499145E-01 -3.62835621E-01 -5.76817081E-01 -6.85316656E-01
 -7.31706067E-01 -2.48833634E-01  3.49067395E-01  1.09263936E-01  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
 -2.55475876E-01 -2.02205607E-01 -8.80305361E-02  0.00000000E+00  0.00000000E+00
  3.70272227E-03  4.39152873E-03  0.00000000E+00  0.00000000E+00 -7.14902837E-03
  0.00000000E+00  2.75749964E-03  3.16317887E-02  5.36551371E-02  6.10688576E-02
  1.48523357E-03 -3.56843804E-02  0.00000000E+00  1.95973754E-02  3.16317887E-02
  5.36551371E-02  6.10688576E-02  1.48523357E-03  3.56843804E-02  0.00000000E+00
  1.95973754E-02
MO  3                     OCC NO =   2.00000000 ORB. ENERGY =  -0.69895149
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  1.58913252E+00
  1.25777631E+00  5.47574938E-01  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  6.31225569E-02  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00 -1.02811543E-01  0.00000000E+00
  0.00000000E+00  0.00000000E+00  5.36226800E-02  9.09569888E-02  1.03524838E-01
  1.32252069E-02 -2.23050502E-02  0.00000000E+00  3.12569803E-02 -5.36226800E-02
 -9.09569888E-02 -1.03524838E-01 -1.32252069E-02 -2.23050502E-02  0.00000000E+00
 -3.12569803E-02
MO  4                     OCC NO =   2.00000000 ORB. ENERGY =  -0.56654332
 -3.91991343E-02 -7.45564300E-02 -1.22103927E-01 -1.94252682E-01 -2.30829601E-01
 -2.47076331E-01 -8.40905597E-02  1.18530085E-01  1.01620061E-01  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  1.76907287E+00  1.40019660E+00  6.09577840E-01  0.00000000E+00  0.00000000E+00
  1.03836552E-01  1.09820683E-02  0.00000000E+00  0.00000000E+00  2.86081107E-02
  0.00000000E+00 -3.95901789E-02 -3.37630486E-02 -5.72702677E-02 -6.51835036E-02
 -5.94631403E-03  3.03934569E-02  0.00000000E+00  7.19304102E-03 -3.37630486E-02
 -5.72702677E-02 -6.51835036E-02 -5.94631403E-03 -3.03934569E-02  0.00000000E+00
  7.19304102E-03
MO  5                     OCC NO =   2.00000000 ORB. ENERGY =  -0.49312113
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  2.04734401E+00  1.62044434E+00  7.05463049E-01
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  1.40893082E-01
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
 -6.98380075E-02  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  2.97221097E-02  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  2.97221097E-02
  0.00000000E+00
END DATA
RHF      ENERGY =        0.0000000000   VIRIAL(-V/T)  =   0.00000000
