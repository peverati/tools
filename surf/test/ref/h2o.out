 #  =====================================================
 # |         DOSURF: QTAIM INTERATOMIC SURFACES          |
 # |    (c) E. Francisco, University of Oviedo, 2017     |
 #  =====================================================
 #
 # DOSURF Calculation starts at lun abr 22 17:45:26 CEST 2019

 # 
 # WARNING: ¡¡¡ ABOUT PARALLELIZATION !!!
 # By now only the OMP model is used. The following 
 # parts of the code of IQA and no others are parallelized :
 # 
 # a) The binary code used to compute the RDM matrices 
 # b) The check of the beta-sphere radii 
 # c) The determination of the atomic surfaces 
 # d) The computation of the angular averages 
 # e) The computation of the bicentric terms 
 # f) The romberg and different reduction like operations 
 # g) The angular averages in the aom task 
 # 
 # Some results can be different when they are compared 
 # with a version that it is not parallelized, this is 
 # because at the end of the regions for which the reduction
 # clause was specified, the original object is updated to 
 # reflect the result of combining its original value with 
 # the final value of each of the private copies using the 
 # operator specified. The reduction operators are all 
 # associative (except for subtraction), and the compiler 
 # may freely reassociate the computation of the final 
 # value. Furthermore, different degrees of parallelism may 
 # result in different numeric results because of changes 
 # in the association of numeric operations. For example, 
 # a serial addition reduction may have a different 
 # pattern of addition associations than a parallel 
 # reduction. These different associations may change the 
 # results of floating-point addition. Usually these 
 # differences are small, near the machine precision, 
 # but can acumulate up to 1d-6/1d-7 differences. 
 # 
 # When using the OMP model, some problems related with 
 # the stack size can occuor, specially with the intel  
 # compiler, to avoid this issue, the following         
 # enviormental variables should be exported, for the   
 # bash shell for example                               
 # 
 # a) export OMP_STACK_SIZE=128m                        
 # b) export KMP_STACK_SIZE=128m                        
 # 

 # Number of OMP threads :   1
 # Number of MPI process :   1
 # 
 # Input number of Primitives       41 reduced to       41
 # Rmaxsurf for all the atoms
    10.000000    10.000000    10.000000
 # CUTOFF for GTOs, eps =  0.1000000000E-14
 # CENTER   1
 # S Shell   Exp =   0.11720000E+05    Cutoff =      0.100000    Primitives :     1
 # S Shell   Exp =   0.17590000E+04    Cutoff =      0.200000    Primitives :     2
 # S Shell   Exp =   0.40080000E+03    Cutoff =      0.300000    Primitives :     3
 # S Shell   Exp =   0.11370000E+03    Cutoff =      0.600000    Primitives :     4
 # S Shell   Exp =   0.37030000E+02    Cutoff =      1.000000    Primitives :     5
 # S Shell   Exp =   0.13270000E+02    Cutoff =      1.700000    Primitives :     6
 # S Shell   Exp =   0.50250000E+01    Cutoff =      2.700000    Primitives :     7
 # S Shell   Exp =   0.10130000E+01    Cutoff =      5.900000    Primitives :     8
 # S Shell   Exp =   0.30230000E+00    Cutoff =     10.700000    Primitives :     9
 # P Shell   Exp =   0.17700000E+02    Cutoff =      1.500000    Primitives :    10   11   12
 # P Shell   Exp =   0.38540000E+01    Cutoff =      3.100000    Primitives :    13   14   15
 # P Shell   Exp =   0.10460000E+01    Cutoff =      5.900000    Primitives :    16   17   18
 # P Shell   Exp =   0.27530000E+00    Cutoff =     11.600000    Primitives :    19   20   21
 # D Shell   Exp =   0.11850000E+01    Cutoff =      5.700000    Primitives :    22   23   24   25   26   27
 # CENTER   2
 # S Shell   Exp =   0.13010000E+02    Cutoff =      1.700000    Primitives :    28
 # S Shell   Exp =   0.19620000E+01    Cutoff =      4.200000    Primitives :    29
 # S Shell   Exp =   0.44460000E+00    Cutoff =      8.900000    Primitives :    30
 # S Shell   Exp =   0.12200000E+00    Cutoff =     16.900000    Primitives :    31
 # P Shell   Exp =   0.72700000E+00    Cutoff =      7.100000    Primitives :    32   33   34
 # CENTER   3
 # S Shell   Exp =   0.13010000E+02    Cutoff =      1.700000    Primitives :    35
 # S Shell   Exp =   0.19620000E+01    Cutoff =      4.200000    Primitives :    36
 # S Shell   Exp =   0.44460000E+00    Cutoff =      8.900000    Primitives :    37
 # S Shell   Exp =   0.12200000E+00    Cutoff =     16.900000    Primitives :    38
 # P Shell   Exp =   0.72700000E+00    Cutoff =      7.100000    Primitives :    39   40   41
 # Total number of shells =           24
 #  24 shells contribute to the basin of center    1
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 #  24 shells contribute to the basin of center    2
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 #  24 shells contribute to the basin of center    3
 # [ shell(atom) means shell number "shell" of atom "atom" ]
      1(   1)     2(   1)     3(   1)     4(   1)     5(   1)     6(   1)     7(   1)     8(   1)
      9(   1)    10(   1)    11(   1)    12(   1)    13(   1)    14(   1)     1(   2)     2(   2)
      3(   2)     4(   2)     5(   2)     1(   3)     2(   3)     3(   3)     4(   3)     5(   3)

 # Surface precision =  0.100000E-04
 # Lebedev grid is used. Angular points follow
 #      1  5810 <--- (Atom & Points)
 #      2  5810 <--- (Atom & Points)
 #      3  5810 <--- (Atom & Points)
 # 
 # RMAXSURF  parameter  =  0.100000E+02
 # DAMPRMAX  parameter  =  0.200000E+01
 # EPSISCP   parameter  =  0.220000E+00
 # SURF will be computed for the following    3 atoms
 #    1   2   3

 # Total elapsed time =       25.074742 seconds
